flexget/plugins/list/series_list.py

```diff
@@ -0,0 +1,283 @@
+from __future__ import unicode_literals, division, absolute_import
+
+import logging
+from collections import MutableSet
+from datetime import datetime
+
+from builtins import *  # pylint: disable=unused-import, redefined-builtin
+from sqlalchemy import Column, Unicode, Integer, ForeignKey, Boolean, DateTime, String, func
+from sqlalchemy.orm import relationship
+from sqlalchemy.sql.elements import and_
+
+from flexget.plugins.filter.series import FilterSeriesBase
```

Comment:
all of them are kind of internal, the main idea is to make coupling harder in order to avoid creating spaghetti mess :)